src/utils/formValidationUtils.js

```diff
@@ -1,11 +1,12 @@
+import { isFinite, isNumber } from "lodash"
+
 export const validateNumberWithinRange = ({
-  min,
-  max,
+  min = Infinity,
+  max = Infinity,
   value,
   errorReturn = false,
-  isOptional = false,
 }) => {
-  if (!value && isOptional) return true
+  if (!isNumber(min) || !isNumber(max) || !isFinite(value)) return errorReturn
```

Comment:
This is done like this so it is optional to include either min or max as in there's no need to include max when you just want to check for min only